packages/user-interface/src/pages/MessageDetailsPage.tsx

```diff
@@ -18,6 +21,25 @@ const MessageDetailsPage = () => {
   });
   const message = messageData?.getBericht as Bericht | undefined;
 
+  if (messageLoading) {
```

Comment:
We removed this loading/error state in the previous MR?

Reply:
You're right. Must've been an erroneus merge on my side, because I started on this before the last PR was approved. I will remove this as the `messageContent` compoennt already handles the state for this.